frappe/installer.py

```diff
@@ -246,7 +246,11 @@ def install_app(name, verbose=False, set_as_patched=True):
 			return
 
 	if name != "frappe":
-		add_module_defs(name)
+		try:
+			add_module_defs(name)
+		except frappe.exceptions.DuplicateEntryError as e:
```

Comment:
btw use `ignore_if_duplicate` instead of catching exception (doesn't work on PG)